src/analysis/PluginBase.py

```diff
@@ -233,10 +239,15 @@ def _update_duration_stats(self, duration):
         if self.analysis_stats_count.value < self.ANALYSIS_STATS_LIMIT:
             self.analysis_stats_count.value += 1
 
-    def _handle_failed_analysis(self, fw_object, process, worker_id, cause: str):
+    def _handle_failed_analysis(  # noqa: PLR0913
+        self, fw_object, process, worker_id, cause: str, trace: str | None = None
```

Comment:
nit: I really hate this formatting. As far as I know there is no option to force black to add the trailing comma..